recipes/hboard-widget/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hboard-widget" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataCanvasIO/HyperBoard/archive/refs/tags/{{ version }}.tar.gz
```

Comment:
Any reason for not using the PyPI release at https://pypi.org/project/hyperboard/#files ?

Reply:
The  https://pypi.org/project/hyperboard/#files  release is not my project. My project is at https://pypi.org/project/hboard-widget/.  Because the name of `hyperboard` already exists, so my project name is `hbboard-widget` instead of `hyperboard-widget `.

conda-forge allows multiple builds in one version, which is more friendly to release. Generally speaking, I want to push to conda-forge first and then to pypi. That's why I use GitHub files instead of pypi files.

Reply:
No problem. Just confusing names :smile: